backend/package.json

```diff
@@ -84,7 +84,8 @@
     "ts-node": "^10.9.2",
     "tsconfig-paths": "4.2.0",
     "typescript": "^5.8.2",
-    "typescript-eslint": "^8.26.1"
+    "typescript-eslint": "^8.26.1",
+    "db": "file:../database"
```

Comment:
the database folder is shared with the backend docker container and you can now npm install local folders instead of the linking.